CMakeLists.txt

```diff
@@ -142,4 +142,21 @@ install(TARGETS ${install_targets} EXPORT evmoneTargets
 install(DIRECTORY ${include_dir}/
     DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
 
+set(CONFIG_PATH ${CMAKE_INSTALL_LIBDIR}/cmake/evmone-assigner)
+install(EXPORT evmoneTargets DESTINATION ${CONFIG_PATH})
+install(EXPORT assignerTargets DESTINATION ${CONFIG_PATH})
```

Comment:
I think, we should use `NAMESPACE` here (it will be added to `IMPORTED` library target). As a result, you will use something like `evm_assigner::evm_assigner` instead of just `evm_assigner`.

The reason for this is that CMake won't treat namespaced target as a "system library" and will immediately fail if the target is not found.
https://stackoverflow.com/a/62490462

The root of the problem is actually a questionable syntax of `target_link_libraries()`, which allows you to pass even linker flags to it.

I believe right now, that using namespaces when exporting targets is the best practice, since in that case you will be sure later, that you are linking exactly this one target rather than some rubbish.